src/lib/containers/download_list_v2/DownloadCartPage.tsx

```diff
@@ -225,7 +225,7 @@ export const DownloadCartPage:React.FunctionComponent<DownloadCartPageProps> = (
                   refetch()
                 }} />}
                 <DownloadListStats numBytes={data.sumOfFileSizesAvailableForDownload} numPackagableFiles={data.numberOfFilesAvailableForDownloadAndEligibleForPackaging} numFiles={data.numberOfFilesAvailableForDownload}/>
-                <AvailableForDownloadTable filesStatistics={data}/> 
+                {refetch && <AvailableForDownloadTable filesStatistics={data} refetchStatistics={refetch}/> }
```

Comment:
The Download Table can now request to refetch the statistics (needed when items are removed from the list, or a package is created).